### ISSUE TITLE
No ACP Menu

### ISSUE THREAD

**Issue by Dynosore** [barisusakli/nodebb-plugin-rss#24]
After reinstalling from ACP, the plugin is fetching content, but there's no route listed. I have already reloaded and restarted NodeBB after installing the plugin to no avail.

![image](https://cloud.githubusercontent.com/assets/8236819/14001779/d21450e4-f115-11e5-869f-e15bb2a2f1cf.png)

**Comment by barisusakli** [comment]
What version? 0.3.5 works fine for me
![image](https://cloud.githubusercontent.com/assets/2583394/14011217/806c256a-f1a5-11e5-81d8-9479055abfa1.png)

**Comment by Dynosore** [comment]
![](http://i.prntscr.com/195662f3e2ac41c1992f9c178fa51330.png)
Yes, I am running 0.3.5

**Comment by Dynosore** [comment]
The issue has apparently resolved itself. I went to restart my server for good measure once more and NodeBB would not start, saying that dependencies were out of date. So I re-ran ./nodebb upgrade (already ran once before when upgrading to 1.02) and then NodeBB was able to start.

After NodeBB restarted successfully, then the RSS plugin was shown. Leaving open in case this is an issue with NodeBB (since I had already ran the upgrade script), but my issue is resolved, and you're free to close this.

**Comment by barisusakli** [comment]
Let me know if it happens again or for another plugin.

**Comment by SemperMMA** [comment]
I'm having the same issue running **Installed 0.3.10 | Latest 0.3.10**.

I've tried going directly to the url (http://***.**/admin/plugins/rss) and receive a 404. 

Ran `npm install` and received this:

`npm WARN package.json nodebb-plugin-rss@0.3.10 No repository field.`

I tried manually installing and uninstalling and receive the same issue.  Tried multiple browsers and no luck. Deactivated all plugins in case there was a conflict and that still didn't work.

Any ideas?
